make_site.py

```diff
@@ -170,36 +172,40 @@ class DocDecl:
     # is simply a link to the right revision of the mathlib source code.
     src_link: str
 
+# keep public fields in sync with `scripts/yaml_check.py` in mathlib4 repo
 @dataclass
 class HundredTheorem:
     number: str
     title: str
-    decl: Optional[str] = None
```

Comment:
Removing this requires changing 100.yaml, imho for the worse. I presume this structure with `decl` and `decls` is to allow nicer syntax for the common case of just a single declaration, which this change would break.

Reply:
Yes, I agree.